dev-server.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  publicPath: 'http://localhost:8080/dist/',
```

Comment:
This touches multiple places and raises a bigger question of installation / config. Let's tackle it later on.